<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRoomInfoDialog</class>
 <widget class="QDialog" name="ChangeRoomInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(180, 230, 255);


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="letRoomNumber">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>房间类型</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="letRoomType">
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
background-color:rgb(237,247,251);
border-radius:5px;
font :18px;
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>房间描述</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="letRoomDepict">
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
background-color:rgb(237,247,251);
border-radius:5px;
font :18px;
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>价格</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="letPrice">
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
background-color:rgb(237,247,251);
border-radius:5px;
font :18px;
}</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>房间状态</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="letStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
QPushButton
{
    color: rgb(255, 255, 255);
background-color: rgb(0,153,255);
border-radius:5px
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnChange">
        <property name="text">
         <string>确定修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
